### PR TITLE
Uppates page and section titles

### LINKED ISSUE #63: Change title on areas vote pages
On areas vote pages:

- the area name is repeated at the top of the page. It's noise that users don't need
- we generally don't use compound titles for pages [section hyphen page name]

It would be better to use:

[election] for the h1

[area name] for the h2

In the example:

"GE 2019" for the h1

"Bexhill and Battle" for the h2

Image

### PR DESCRIPTION
**Pull request by markconroy**
Closes #63

### CODE REVIEW

File: localgov_elections.module [hunk 1]
```diff
@@ -826,6 +826,16 @@ function localgov_elections_preprocess_block(&$variables) {
...
+  // edit form, so it's easier to find on the /admin/content page.
+  $node = \Drupal::request()->attributes->get('node');
+  if ($node && $node->bundle() == 'localgov_area_vote') {
+    if ($variables['base_plugin_id'] == 'localgov_page_header_block') {
+      $variables['content'][0]['#title'] = $node->get('localgov_election_area_name')->value;
+    }
+  }
 }
 
 /**
```
Comment (chriswales95) [inline localgov_elections.module]: This throws an error on view pages clicked from the menu block. $node is not always an entity it is sometimes a string.

**Review by chriswales95 — CHANGES_REQUESTED** [review]
Hi @markconroy, seems good but one issue I think we need to address!

File: localgov_elections.module (hunk 1 above)
Comment (chriswales95) [reply localgov_elections.module]: Well, at least on my version of Drupal. This fixed it locally for me:

```
if (!$node){
    return;
}
 
if (!($node instanceof NodeInterface)){
    $node = Node::load($node);
}
```

**Comment by finnlewis** [comment]
@chriswales95 are you happy with this one now?

**Review by chriswales95 — APPROVED** [review]
Looks good to me now. No errors.

**Pull request merged**